src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -222,6 +225,14 @@ public Optional<AdditionalNotes> getAdditionalNotes() {
             return Optional.ofNullable(additionalNotes);
         }
 
+        public void setClass(Class classDateTime) {
+            this.classDateTime = classDateTime;
+        }
+
+        public Optional<Class> getAClass() {
```

Comment:
Could you explain why the method is named as `getAClass()` instead of `getClass()`? And why `setClass()` is named this way instead of `setAClass()`?

Reply:
I see. This makes much sense to me!